### ISSUE TITLE
NVDA does not export all errors in LibreOffice Writer.

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#19326]
### Brief summary

Write a text with several errors in Writer.
Make mistakes with accents, missing capital letters, inverted capital letters, misspelled words, missing punctuation. Make several mistakes in one line.

### Steps to reproduce

Write a text with several errors in Writer.
Make mistakes with accents, missing capital letters, inverted capital letters, misspelled words, missing punctuation. Make several mistakes in one line.

### Actual behavior

When reading the document with the arrows above and below, only some errors are detected by NVDA.
I tried to notice any correlation, but it seems to vary; that is, sometimes the reported error is the first on the line, sometimes it's the second on the line.
Sometimes the reported error is due to the use of the wrong letter, other times it was an accentuation error. Similarly, accentuation errors were not detected at all other times.

The writer is correctly highlighting the incorrect word, because even if NVDA doesn't announce the error, when pressing Applications (right-click), the correction suggestion is shown.

### Expected behavior

all errors are announced

### NVDA logs, crash dumps and other attachments

[tudo errado.docx](https://github.com/user-attachments/files/23916288/tudo.errado.docx)

### NVDA type

installed copy

### NVDA version

nvda.exe, NVDA alpha-53725,7ba333a8

### Have you tried any other versions of NVDA? If so, please report their behaviors.

no

### Windows version

Windows 11 25H2 (AMD64) construir 26200.7171

### Name and version of other software in use when reproducing the issue

libre office writer 26.2 dev

### Other information about your system

The text I attached is in Brazilian Portuguese, with several errors; the entire system is in Portuguese.

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have not run the COM Registration Fixing Tool

**Comment by Adriani90** [comment]
@fernando-jose-silva did you test with a document that has been created natively in libreeoffice writer, file format .odt? Does that issue still occur? Or does it occur only when you open .docx files in LO Writer?

**Comment by seanbudd** [comment]
Related #19155 cc @michaelweghorn

**Comment by fernando-jose-silva** [comment]
This problem occurs even in documents created directly in Writer.
Some findings:

The word "residencia" in Portuguese seems to have problems; sometimes Writer displays it as incorrect when written without the accent "ê," other times it doesn't. I think that's more of a Writer bug.
The word "numero" written in Brazilian Portuguese without the accent "ú" is not highlighted as incorrect by Writer.
Word's spell checker catches all of this.
Grammatical errors like capitalization are not reported by NVDA, so that's not a bug; it would be a new feature.
So, if someone more knowledgeable than me could point out similar errors, I think we would have to open a new ticket suggesting that NVDA detect grammar errors in Writer.

**Comment by michaelweghorn** [comment]
@fernando-jose-silva Can you please give exact steps to reproduce this for one case in the sample document for which a spelling error is visually displayed on screen, but not announced by NVDA, like

1. open sample document https://github.com/user-attachments/files/23916288/tudo.errado.docx
2. ... (further steps, e.g. press this and that key,...)

**Comment by fernando-jose-silva** [comment]
Set the LibreOffice Writer spell checker to Brazilian Portuguese.

Open the attached document tudo errado.docx.
Press the down arrow to read the second line:
My house is located on Uniflor street, number 21.
Minha caza fica na rua Uniflor numero 21
Note that before the word "number numero," NVDA does not announce a spelling error.
It is expected that it will announce a spelling error for "number numero."
Use Ctrl + right arrow to navigate to the word "number numero" and press Applications. Notice that LibreOffice Writer suggests the correction "number número" for the misspelled word, so Writer recognizes the misspelled word "number número," but NVDA does not announce a spelling error for "number número" when reading the line.
Press the down arrow to read the third 3 line:
To go to my residence, I take 2 buses and a subway.
Para ir para minha residencia pego 2 onibus e um metrô.
Note that NVDA displays the word "residence residencia" without indicating a spelling error beforehand.

The expected thing is to hear
spelling error residencia
walk word by word on line 3 until you find the word residencia, press applications and see that the writer suggests residencia which is the correction for the word, which means that the writer recognized the error but NVDA did not announce it when navigating and reading line by line.

[tudo errado.docx](https://github.com/user-attachments/files/24057362/tudo.errado.docx)
